README.md

```diff
@@ -297,7 +297,7 @@ const gauge = new client.Gauge({
 // 1st version: Set value to 100 with "method" set to "GET" and "statusCode" to "200"
 gauge.set({ method: 'GET', statusCode: '200' }, 100);
 // 2nd version: Same effect as above
-gauge.labels('GET', '200').set(100);
```

Comment:
I think we should still keep this in the docs, but perhaps below the object one?

Reply:
yes I agree, I will do it soon

Reply:
done